Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs

```diff
@@ -255,6 +255,12 @@ public void Set(MixedRealityRaycastHit hit, RayStep ray, int rayStepIndex, float
             public void Set(RaycastResult result, Vector3 hitPointOnObject, Vector4 hitNormalOnObject, RayStep ray, int rayStepIndex, float rayDistance)
             {
                 raycastHit = default(MixedRealityRaycastHit);
+                raycastHit.point = hitPointOnObject;
+                raycastHit.normal = hitNormalOnObject;
+                raycastHit.distance = rayDistance;
+                raycastHit.transform = result.gameObject.transform;
+                raycastHit.raycastValid = false;
```

Comment:
```suggestion
                raycastHit.raycastValid = false; //we can't populate all fields of MixedRealityRaycastHit in case of a GraphicsRaycast so we flag the hit data as invalid 
```